tools/load-generator/main.go

```diff
@@ -139,9 +139,10 @@ func (q *Querier) run(wg *sync.WaitGroup) {
 	}
 }
 
+// understand from here query function
 func (q *Querier) query(expr string) {
 	queryCount.WithLabelValues(q.target, q.name, expr, q.qtype).Inc()
-	start := time.Now()
+	start := time.Date(2023, time.September, 15, 10, 30, 0, 0, time.UTC)
```

Comment:
This looks ok for the disk-block queries, but we will also need queries against the head-block, i.e. relative to 'now'. 
I haven't looked in detail, but suggest you make this part of the query group config.

Reply:
actually i have reviste this part , i will look into this today.